smithy-codegen-core/src/main/java/software/amazon/smithy/codegen/core/trace/TraceFile.java

```diff
@@ -226,8 +225,8 @@ public String getSmithyTrace() {
      *
      * @return an ArtifactMetadata object.
      */
-    public ArtifactMetadata getArtifactMetadata() {
-        return artifactMetadata;
+    public TraceMetadata getTraceMetadata() {
```

Comment:
I think this should just be `getMetadata` to match the file format. The class name doesn't have to have a major impact on the getter name.